setup.py

```diff
@@ -29,22 +29,22 @@ def read_description():
 setup(
     name='nafas',
     packages=['nafas'],
-    version='0.1',
+    version='0.2',
     description='Breathing gymnastics application',
     long_description=read_description(),
     long_description_content_type='text/markdown',
     author='Sepand Haghighi',
     author_email='sepand@pyrgg.ir',
     url='https://github.com/sepandhaghighi/nafas',
-    download_url='https://github.com/sepandhaghighi/nafas/tarball/v0.1',
+    download_url='https://github.com/sepandhaghighi/nafas/tarball/v0.2',
     keywords="python3 python breath breathing meditation",
     project_urls={
         'Source': 'https://github.com/sepandhaghighi/nafas',
     },
     install_requires=get_requires(),
     python_requires='>=3.5',
     classifiers=[
-        'Development Status :: 1 - Planning',
+        'Development Status :: 3 - Alpha',
```

Comment:
Isn't it `2`?

Reply:
No that's `Development Status :: 2 - Pre-Alpha`